Week-10/shortly/src/components/Navbar/index.tsx

```diff
@@ -0,0 +1,165 @@
+import * as React from 'react'
+
+import AdbIcon from '@mui/icons-material/Adb'
+import AppBar from '@mui/material/AppBar'
+import Avatar from '@mui/material/Avatar'
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import Container from '@mui/material/Container'
+import IconButton from '@mui/material/IconButton'
+import Menu from '@mui/material/Menu'
+import MenuIcon from '@mui/icons-material/Menu'
+import MenuItem from '@mui/material/MenuItem'
+import Toolbar from '@mui/material/Toolbar'
+import Tooltip from '@mui/material/Tooltip'
+import Typography from '@mui/material/Typography'
+
+const pages = ['Features', 'Pricing', 'Resources']
+const settings = ['Profile', 'Account', 'Dashboard', 'Logout']
+
+const Navbar = () => {
```

Comment:
@apandyamob Since you have used type for each state variable, i think you can make it a full typescript application. Follow all react types as well. For eg: React.FC etc

Good to have

Reply:
added React.FC type